create_aio_app/utils/check-requirements.py

```diff
@@ -0,0 +1,28 @@
+import subprocess
+
+uniq_tools = ""
+
+with open("requirements-dev.txt", "r") as f:
```

Comment:
In this case u can iterate direcory's files

```python
import os
from pathlib import Path      

path = Path('./requirements') 

for file in os.listdir(path):
    ... 
```

Reply:
I believe `pathlib` has built-in methods for that.